spec/controllers/admin/user_creations_controller_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Admin::UserCreationsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+
+  describe "GET #hide" do    
```

Comment:
It looks like `hide` and `set_spam` are both `PUT` actions, if you could update the specs and descriptions to reflect that https://github.com/otwcode/otwarchive/blob/148004d1f4c29f4ee4f31a2f25633e06e0dff63a/config/routes.rb#L169-L174